### DISCUSSION TITLE
guard breaks can happen on just defense

### DISCUSSION THREAD

**Discussion by D-MAS** [ikemen-engine/Ikemen-GO#1682] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

i just vehemently disagree with this and am too much of a scrub to learn actual programming, change it. or don't.

### Search terms

guardbreak, just defense, just guard, just defend

**Comment by potsmugen** [comment, marked as answer]
The just defense message is more like a placeholder to see that you can do it than anything. The window is very tight for that reason. Your idea has merit but doing it that way feels like trying to design a full game rather than giving people the base to do it themselves, if that makes sense.

**Comment by shinra358** [comment]
yeah there is no game where jd contributes to the degradation of the guard meter

**Reply by D-MAS** [reply]
yes, might actually code it myself out of sheer spite now
